pkg/action/diff.go

```diff
@@ -153,10 +154,12 @@ func fileDestination(ctx context.Context, c bincapz.Config, fr, tr *bincapz.File
 	}
 
 	// are there already modified behaviors for this file?
-	if _, exists := d.Modified[relPath]; !exists {
-		d.Modified[relPath] = abs
+	if _, exists := d.Modified.Get(relPath); !exists {
+		d.Modified.Set(relPath, abs)
 	} else {
-		d.Modified[relPath].Behaviors = append(d.Modified[relPath].Behaviors, abs.Behaviors...)
+		rel, _ := d.Modified.Get(relPath)
+		rel.Behaviors = append(rel.Behaviors, abs.Behaviors...)
+		d.Modified.Set(relPath, rel)
```

Comment:
This was the most difficult portion of migrating to `orderedmap`. I don't think a one-liner was doable but this is more readable.